test/micro/org/openjdk/bench/java/lang/foreign/HeapMismatchManualLoopTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ *  Copyright (c) 2020, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should probably also be 2025?
```suggestion
 *  Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```